style.css

```diff
@@ -0,0 +1,68 @@
+
+header {
+    background: #5e5e5e;
+    padding: 1em;
+    display: flex;
+}
+header a {
+    text-decoration: none;
+    color: white;
+    margin-right: 1em;
+}
+.main {
+    display: flex;
+    flex-direction: column;
+    padding: 2em 20vw;
+}
+h2 {
+    display: block;
+    font-size: 1.5em;
+    margin-block-start: 0.83em;
```

Comment:
Never seen use case for margin-block-start. If you want to work on basic spacing of an element normal margin and padding attributes will suit you better.

ref: 
- https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
It is better to use margin or padding attribute, also you can check on this web page which browser supports those properties.
https://caniuse.com/#search=margin-block-start